### ISSUE TITLE
OOM received in SharedClasses.SCM01.MultiThread on internal machine

### ISSUE THREAD

**Issue by Mesbah-Alam** [adoptium/aqa-systemtest#253]
OOM is received in SharedClasses.SCM01.MultiThread while it tries to run one of its jvm sub-processes, on internal Redhat machine `rhel6x64rt1-1` 

Crash output (cropped): 
```
MT3 stderr JVMDUMP039I Processing dump event "systhrow", detail "java/lang/OutOfMemoryError" at 2019/04/12 04:10:24 - please wait.
MT3 stderr JVMDUMP032I JVM requested System dump using '/home/jenkins/workspace/Test_openjdk8_j9_extended.system_x86-64_linux_Nightly/openjdk-tests/TestConfig/test_output_15550658279207/SharedClasses.SCM01.MultiThread_0/20190412-041021-SharedClasses/results/core.20190412.041024.27083.0001.dmp' in response to an event
MT3 stderr JVMDUMP012E Error in System dump: insufficient system resources to generate dump, errno=11 "Resource temporarily unavailable"
MT5 stderr JVMDUMP039I Processing dump event "systhrow", detail "java/lang/OutOfMemoryError" at 2019/04/12 04:10:24 - please wait.
STF 04:10:24.391 - Found dump at: /home/jenkins/workspace/Test_openjdk8_j9_extended.system_x86-64_linux_Nightly/openjdk-tests/TestConfig/test_output_15550658279207/SharedClasses.SCM01.MultiThread_0/20190412-041021-SharedClasses/results/heapdump.20190412.041024.27083.0002.phd
```

Note: 

1) Adding `-Xmx128m` limit to each of the jvm subprocesses still produces the OOM on `rhel6x64rt1-1`.
2) Adding `-Xmx256m` limit does **not** produce the OOM on  `ub16x64rt2-1`
(Links shared with Joe on Slack)

Does `rhel6x64rt1-1` have its ulimit set to a really small value? If yes, we probably should increase it. The test should have an `-Xmx` limit set on its sub-processes regardless.

**Comment by Mesbah-Alam** [comment]
@jdekonin

**Comment by jdekonin** [comment]
Seems the default `max user processes (-u)` on our redhat6 instances is `1024`.  The ub16 instance is defaulting to `64023`.  I can get a passing `SharedClasses.SCM01.MultiThread` job if I increase this value.

Is there a baseline ulimit values documented for what is required?

**Comment by Mesbah-Alam** [comment]
There is no baseline ulimit value documented anywhere for systemtest (@lumpfish - are you aware of one?).  I suggest we come up with a baseline value based on our findings in this issue.

**Comment by lumpfish** [comment]
There is nothing documenting the ulimit 'requirement' to run the system tests, individually or collectively.  Up until this was raised the tests had been running successfully with the values already set on the test machines, though I'm not surprised that -u 1024 is too small.
_"I suggest we come up with a baseline value based on our findings in this issue."_
That seems a sensible approach.

**Comment by lumpfish** [comment]
This should now be fixed via https://github.com/AdoptOpenJDK/openjdk-tests/pull/2274

**Comment by karianna** [comment]
> This should now be fixed via [AdoptOpenJDK/openjdk-tests#2274](https://github.com/AdoptOpenJDK/openjdk-tests/pull/2274)

\safe to close this?
